transactron/lib/reqres.py

```diff
@@ -63,8 +65,8 @@ def __init__(self, args_layout: MethodLayout, results_layout: MethodLayout, src_
             Alternatively, the source location to use instead of the default.
         """
         self.src_loc = get_src_loc(src_loc)
-        self.results_layout = results_layout
-        self.args_layout = args_layout
+        self.results_layout = from_method_layout(results_layout)
```

Comment:
method layout -> Layout in docstring

Reply:
The `method layout` wording is used in other places, including the core. I think this is outside the scope of the PR.